views/home/viewreference.ejs

```diff
@@ -49,6 +49,16 @@
             </div>
           </div>
         </div>
+        <div ng-hide="selectedRef.privatenotes===null" class="col-md-6">
```

Comment:
Spaces here also :-)

Also, maybe do ng-if rather than ng-hide. ng-if means we don't have things in the DOM that we won't be using. It's much nicer in almost every case.